pkg/api/v1beta1/types.go

```diff
@@ -170,9 +170,11 @@ type GitRepoVolumeSource struct {
 }
 
 // SecretVolumeSource adapts a Secret into a VolumeSource
+//
+// https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md
 type SecretVolumeSource struct {
-	// Reference to a Secret
-	Target ObjectReference `json:"target" description:"target is a reference to a secret"`
+	// Name of the secret in the pod's namespace to use
+	Target string `json:"target" description:"target is the name of a secret in the pod's namespace"`
```

Comment:
This is an incompatible change.  v1b[12] should stay as ObjectRef unless you're REALLY confident that nobody anywhere is using this...